raiden/network/rpc/client.py

```diff
@@ -446,21 +455,18 @@ def poll(self, transaction_hash: bytes, confirmations: int = None):
 
             # the transaction was added to the pool and mined
             if transaction and transaction['blockNumber'] is not None:
-                break
+                last_result = transaction
 
-            last_result = transaction
-            gevent.sleep(.5)
+                # this will wait for both APPLIED and REVERTED transactions
+                transaction_block = transaction['blockNumber']
+                confirmation_block = transaction_block + self.default_block_num_confirmations
 
-        if confirmations:
-            # this will wait for both APPLIED and REVERTED transactions
-            transaction_block = transaction['blockNumber']
-            confirmation_block = transaction_block + confirmations
+                block_number = self.block_number()
 
-            block_number = self.block_number()
+                if block_number >= confirmation_block:
+                    return transaction
 
-            while block_number < confirmation_block:
-                gevent.sleep(.5)
-                block_number = self.block_number()
+            gevent.sleep(1.0)
```

Comment:
Why did you increase the polling timeout to 1 second from 0.5 here?

Reply:
With average block times of 15s it doesn't make sense to poll every 0.5s. Especially as we now use `getTransaction` and not only `blockNumber` in the polling, this also helps reducing load on the Ethereum node.

Reply:
All right. I don't disagree -- just wanted to see your way thinking. Can you perhaps put this as  a constant variable somewhere. I dislike magic numbers.